system_test/helpers/aest_docker_api.erl

```diff
@@ -144,6 +145,17 @@ kill_container(ID) ->
             throw({docker_error, maps:get(message, Response)})
     end.
 
+wait_stopped(ID, Timeout) ->
+    Query = #{condition => "not-running"},
+    PostOpts = #{timeout => Timeout},
+    case docker_post([containers, ID, wait], Query, #{}, PostOpts) of
+        {error, timeout} -> timeout;
```

Comment:
Same as `error` above, I don't see any code checking this. Should be a throw instead?

Reply:
The timeout result is actually used in `aest_docker:291`.

Reply:
Ah, missed that. Looks okay to me